docs/commands/super.md

```diff
@@ -774,25 +787,32 @@ WHERE id LIKE '%in case you have any feedback 😊%'
   OR payload.member.type LIKE '%in case you have any feedback 😊%'
 ```
 There are 486 such fields.  You can review the entire query in
-[docs/commands/search.sql](search.sql).
+[`search+.sql`](https://github.com/brimdata/super/blob/main/scripts/super-cmd-perf/search%2B.sql).
```

Comment:
Since the scripts currently only exist on this branch, this hyperlink will fail until this PR is merged.